exercises/concept/log-levels/.meta/exemplar.py

```diff
@@ -32,14 +42,31 @@ def parse_log_level(message):
 
 
 def convert_to_short_log(log_level, message):
+    """Converts a log message to its shorter format.
+
+    :param log_level: enum - 'LogLevel.<level>'  e.g.  'LogLevel.Error'
+    :param message: str - log message
+    :return: enum -  'LogLevelInt.<value>` e.g. 'LogLevelInt.5'
+    """
+
     return f'{LogLevelInt[log_level.name].value}:{message}'
 
 
 def get_warn_alias():
+    """Returns the enum for LogLevel.Warning.
```

Comment:
Docstrings should be imperative ("Do something ..." or "Return ...") vs descriptive ("Does sometime ..." or "Returns ...").

This file has some of both. It would be nice to make all the docstrings in this exercise consistent.